packages/env-tests/src/monorepoRun.ts

```diff
@@ -3,7 +3,7 @@ import Web3 from 'web3'
 import { loadFromEnvFile } from './env'
 import { rootLogger } from './logger'
 import { clearAllFundsToRoot, parseStableTokensList } from './scaffold'
-import { runAttestationTest } from './tests/attestation'
+// import { runAttestationTest } from './tests/attestation'
```

Comment:
nit: any reason not to delete instead of commenting out?

Reply:
I left it as well as the `attestation` file just in case we wanted to go back and update them for ASv2 in the future, but probably makes more sense to just delete